src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,13 +43,14 @@
     
     private Joystick driverStick;
     private Joystick controlStick;
-    private AnalogSelector analogSelectorOne;
+    public static AnalogSelector analogSelectorOne;
```

Comment:
Any reasoning for making this public static?

Reply:
This is being used in the selector test code (line 84) in  Robot.java

Reply:
I think we should remove the debug statement from Robot.java. If you think it is needed maybe we can think of a more object oriented way to share this variable.

Reply:
I set it back to private.  It was previously public because it was being used for the test code which I now removed.